packages/node-common/src/logger.ts

```diff
@@ -0,0 +1,79 @@
+// @ts-ignore
```

Comment:
This is kind of dumb but I had to do it because we can't use anything that uses the `fs` package in ios or android.  We have to split it up amongst the desktop/node environments and mobile.